cmd/stdiscosrv/apisrv.go

```diff
@@ -229,7 +231,17 @@ func (s *apiSrv) handleGET(ctx context.Context, w http.ResponseWriter, req *http
 func (s *apiSrv) handlePOST(ctx context.Context, remoteAddr *net.TCPAddr, w http.ResponseWriter, req *http.Request) {
 	reqID := ctx.Value(idKey).(requestID)
 
-	rawCert := certificateBytes(req)
+	rawCert, err := certificateBytes(req)
```

Comment:
This is the same as the following `if`, apart from the log line. Perhaps they  could be combined?

Reply:
yes, could remove `if rawCert == nil` statement.